### DISCUSSION TITLE
Laravel Reverb in Livewire V3

### DISCUSSION THREAD

**Discussion by ali-hasnain-dev** [livewire/livewire#8832] [1. Help]
Hello everyone,

I'm getting issue while using laravel reverb it is not working properly you can see in attached files

![laravel reverb 2](https://github.com/user-attachments/assets/74b54a5f-ef52-43b3-87ca-d90fd6bb80f2)
![laravel reverb 3](https://github.com/user-attachments/assets/121ac1ab-ef5b-4fd5-97f1-216067553c64)
![laravel reverb1](https://github.com/user-attachments/assets/5a1cc463-04ba-4c99-9b5a-dbc43f0f6a9d)

**Comment by Fulue** [comment]
In the screenshot I see that tailwind is connected via CDN, maybe you are not connected app.js ?
`<script src="{{ mix('js/app.js') }}" defer></script>`

**Reply by ali-hasnain-dev** [reply]
In my main project I'm facing same issue you can see attach image of app.blade.php
![laravel reverb 4](https://github.com/user-attachments/assets/f7852143-68b4-47db-9a39-cb4459c964c8)

**Comment by mojjysolo1** [comment]
change  @vite('resources/css/app.css') to  @vite('resources/js/app.js')
